libcontainer/system/proc.go

```diff
@@ -44,10 +46,10 @@ func (s State) String() string {
 	}
 }
 
-// Stat_t represents the information from /proc/[pid]/stat, as
+// StatT represents the information from /proc/[pid]/stat, as
```

Comment:
I guess the name `Stat_t` was used as it resembles C / Unix `stat_t` type (which has `_t` suffix to distinguish it from `stat()` function), and is used here for the same reason (to distingush from Stat() function).

I like Stat_t much better, despite its non-conformance to Go standards. Note that golang.org/x/sys/unix (as well as stdlib's syscall package) also have names like `Stat_t` and `Statfs_t`.

Reply:
Surely we can just rename it to e.g. ProcStat or something.